### ISSUE TITLE
.NET Core app fails to send otlp data to otel collector 

### ISSUE THREAD

**Issue by tonyqus** [SigNoz/signoz#254]
Hi Team,

I'm working on making tracing of .NET core app work on Signoz. I've added the code of OpenTelemetry with OTLP exporter [here](https://github.com/tonyqus/sample-dotnetcore-app/blob/cf9372e951d51f72089c807377222b490984fae4/Src/WebUI/Startup.cs#L75)

And I started the docker-compose on a Windows laptop as below:
![image](https://user-images.githubusercontent.com/772561/129460454-f11990a2-51d1-4539-8b5e-9331019188f9.png)

I setup the EndPoint of otel receiver as http://<my machine name>:55681/v1/trace. Is this correct?

I figure this out by the following settings in docker-compose.yaml
```
    ports:
      - "1777:1777"   # pprof extension
      - "8887:8888"   # Prometheus metrics exposed by the agent
      - "14268:14268"       # Jaeger receiver
      - "55678"       # OpenCensus receiver
      - "55680:55680"       # OTLP HTTP/2.0 legacy port
      - "55681:55681"       # OTLP HTTP/1.0 receiver
      - "4317:4317"       # OTLP GRPC receiver
      - "55679:55679" # zpages extension
      - "13133"       # health_check
```

However, I still cannot make it work. 

The otel collector always returns me HTTP 501 code with Json data 
`{code=12, message='Not Implemented'}`.

Any idea?

**Comment by ankitnayan** [comment]
@tonyqus try updating url to `http://<SigNoz_IP>:4317`. OTLP exporter only supports `grpc` as mentioned in [this](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1923#issuecomment-803192453).

**Comment by prashant-shahi** [comment]
[OTLP HTTP Exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/v0.43.0/exporter/otlphttpexporter) is released in `v0.43.0` on port `4318` but still marked as "Beta".

Here is the sample configuration to enable it:
```
exporters:
  otlphttp:
    endpoint: https://example.com:4318/v1/traces
```
